src/changelog-notes/default.ts

```diff
@@ -73,35 +73,38 @@ export class DefaultChangelogNotes implements ChangelogNotes {
       this.headerPartial || preset.writerOpts.headerPartial;
     preset.writerOpts.mainTemplate =
       this.mainTemplate || preset.writerOpts.mainTemplate;
-    const changelogCommits = commits.map(commit => {
-      const notes = commit.notes
-        .filter(note => note.title === 'BREAKING CHANGE')
-        .map(note =>
-          replaceIssueLink(
-            note,
-            context.host,
-            context.owner,
-            context.repository
-          )
-        );
-      return {
-        body: '', // commit.body,
-        subject: htmlEscape(commit.bareMessage),
-        type: commit.type,
-        scope: commit.scope,
-        notes,
-        references: commit.references,
-        mentions: [],
-        merge: null,
-        revert: null,
-        header: commit.message,
-        footer: commit.notes
-          .filter(note => note.title === 'RELEASE AS')
-          .map(note => `Release-As: ${note.text}`)
-          .join('\n'),
-        hash: commit.sha,
-      };
-    });
+    const changelogCommits = commits
+      // Filter out commits that are just release commits, they shouldn't be part of the changelog
+      .filter(commit => !commit.bareMessage.includes('chore: release '))
```

Comment:
minor nit: should this be `.startswith` instead of `.includes`? Does this matter?